Client/Frontend/Settings/Main/Account/ConnectSetting.swift

```diff
@@ -0,0 +1,32 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+
+// Sync setting for connecting a Firefox Account. Shown when we don't have an account.
+class ConnectSetting: WithoutAccountSetting {
+    override var accessoryView: UIImageView? { return SettingDisclosureUtility.buildDisclosureIndicator(theme: theme) }
```

Comment:
nit: Break for long line
```suggestion
    override var accessoryView: UIImageView? { 
        return SettingDisclosureUtility.buildDisclosureIndicator(theme: theme) 
    }
```